site/gatsby-site/i18n/locales/es/popovers.json

```diff
@@ -0,0 +1,54 @@
+{
+    "title": {
+        "title": "Título",
+        "text": "La mayoría de los trabajos tienen un título que debe ingresarse aquí palabra por palabra. En caso de que el trabajo no tenga título, intenta resumir el contenido en unas pocas palabras sin editorializar. Este texto se destaca de manera destacada en los registros de incidentes"
```

Comment:
Change `Este texto se destaca de manera destacada en los registros de incidentes` to `Este texto ocupa un lugar destacado en los registros de incidentes.`